src/main/java/toughasnails/config/GameplayConfigurationHandler.java

```diff
@@ -8,26 +8,131 @@
 package toughasnails.config;
 
 import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.FilenameUtils;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
 
 import net.minecraftforge.common.config.Configuration;
 import net.minecraftforge.fml.client.event.ConfigChangedEvent;
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
 import toughasnails.api.config.GameplayOption;
 import toughasnails.api.config.SyncedConfig;
 import toughasnails.core.ToughAsNails;
+import toughasnails.temperature.BlockTemperatureData;
+import toughasnails.temperature.BlockTemperatureStateData;
+import toughasnails.temperature.MaterialTemperatureData;
 
 public class GameplayConfigurationHandler
 {
     public static final String SURVIVAL_SETTINGS = "Survival Settings";
     
     public static Configuration config;
     
+    public static HashMap<String, BlockTemperatureData> blockTemperatureData;
+    
+    public static MaterialTemperatureData materialTemperatureData;
+    
     public static void init(File configFile)
     {
         if (config == null)
         {
             config = new Configuration(configFile);
             loadConfiguration();
+            
```

Comment:
All of the json stuff should really go in a seperate class file. GameplayConfigurationHandler is intended specifically for gameplay.cfg